frontend/package.json

```diff
@@ -11,15 +11,19 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "healthcare-shared": "file:shared",
-    "@reduxjs/toolkit": "1.5.0",
+    "@hookform/resolvers": "^1.3.5",
```

Comment:
Let's install the package only with the fixed version
You need do reinstall the package with `-E` flag. For example `npm i @hookform/resolvers -SE`
More about the exact flag: https://docs.npmjs.com/cli/v7/commands/npm-install